README.md

```
@@ -1,67 +1,125 @@
+---
 
 ## Installation
 
-Add our [NuGet](https://www.nuget.org/packages/FreakyEffects) package or
+```
+dotnet add package FreakyEffects
+```
+
+Or via Package Manager Console:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)